server/opendp_apps/dataverses/views.py

```diff
@@ -96,3 +105,59 @@ def view_as_dict(request, object_id):
         return JsonResponse(get_json_error('Object not found'), status=HTTPStatus.NOT_FOUND)
 
     return JsonResponse(get_json_success('Success', data=mparams.as_dict()))
+
+
+class DataverseUserView(APIView):
+
+    def post(self, request):
+        opendp_user_id, dataverse_handoff_id = request.POST['user_id'], request.POST['dataverse_id']
```

Comment:
Can you add the "user_id" and "dataverse_id" strings to the `dataverses/static_vals.py` file
Add error handling if these keys (user_id, dataverse_id) don't exist